messages/dateinput/dateinput.ar.yml

```diff
@@ -5,3 +5,6 @@ kendo:
 
     # The label for the **Decrement** button in the DateInput
     decrement: تخفيض القيمة
+
+    # The title of the **Clear** button in the DateInput
+    decrement: Clear
```

Comment:
The message key should be changed to `clear` or respectively to how it is going to be used in the component :)